appmgr/src/net/wifi.rs

```diff
@@ -175,23 +175,22 @@ pub async fn delete(#[context] ctx: RpcContext, #[arg] ssid: String) -> Result<(
 #[derive(serde::Serialize, serde::Deserialize)]
 #[serde(rename_all = "kebab-case")]
 pub struct WiFiInfo {
-    ssids: Vec<String>,
-    connected: Option<String>,
```

Comment:
More descriptive, also pair each ssid with the strength instead of for all. The signal strength was valid for just the connected.